website/project/views/contributor.py

```diff
@@ -292,7 +292,7 @@ def project_manage_contributors(auth, node, **kwargs):
         )
         if node.is_public:
             return {'redirectUrl': node.url}
-        return {'redirectUrl': web_url_for('dashboard')}
+        return {'redirectUrl': web_url_for('my_projects')}
```

Comment:
I tried to keep what it was before. The old code `web_url_for('dashboard')` before targets the `/myprojects` page. But you are right, I will ask Sara and Niki on what they think.